src/motion/actuators/GenericActuator.hx

```diff
@@ -324,6 +334,37 @@ class GenericActuator<T> implements IGenericActuator {
 	}
 	
 	
+	/**
+	 * Defines a function which will be called when the tween finishes playing reversed
+	 * @param	handler		The function you would like to be called
+	 * @param	parameters		Parameters you would like to pass to the handler function when it is called
+	 * @return		The current actuator instance
+	 */
+	public function onReverseComplete (handler:Dynamic, parameters:Array <Dynamic> = null):GenericActuator<T> {
+		
+		_onReverseComplete = handler;
+		
+		if (parameters == null) {
+			
+			_onReverseCompleteParams = [];
```

Comment:
I get that this code is a copy-paste from other `onWhatever` methods, and in general Actuate is not super good when it comes to allocations and performance, but it still bugs me and maybe this is something we can improve in future: this null check is not really needed, because `callMethod` does the null check again anyway, so we can probably remove the checks in all `onWhatever` methods.